lib/dark_worlds_server/matchmaking/matching_coordinator.ex

```diff
@@ -54,6 +55,7 @@ defmodule DarkWorldsServer.Matchmaking.MatchingCoordinator do
     bot_count = @session_player_amount - length(state.players)
     {:ok, game_pid, game_config} = start_game(bot_count)
```

Comment:
Since this change will remove the responsibility of starting the bots from the runner i think we should remove this "bot_count" parameter from the runner initialization. 
This change implies changing:
`start_game/1` 
`RunnerSupervisor.start_child/1`
`Runner.init/1`